src/wp-includes/comment-template.php

```diff
@@ -2309,10 +2309,19 @@ function comment_form( $args = array(), $post_id = null ) {
 			 */
 			do_action( 'comment_form_must_log_in_after' );
 		else : ?>
-			<form action="<?php echo esc_url( $args['action'] ); ?>" method="post" id="<?php echo esc_attr( $args['id_form'] ); ?>" class="<?php echo esc_attr( $args['class_form'] ); ?>"<?php echo $html5 ? ' novalidate' : ''; ?>>
+			<form action="<?php echo esc_url( $args['action'] ); ?>" method="post" id="<?php echo esc_attr( $args['id_form'] ); ?>" class="<?php echo esc_attr( $args['class_form'] ); ?>"<?php echo $html5 ? ' novalidate' : ''; ?>
+			<?php
+			/**
+			 * Fires inside the comment form tag.
+			 *
+			 * @since CP-1.x.x
+			 */
+			do_action( 'comment_form_tag', $args );
```

Comment:
I don't think an action is the best usage here. It would be better to filter the form tag, and even better to use the new attribute filter function in #872.

Reply:
Ha! Interesting that you say that, because I was originally looking for a filter. But this is how WP and CP already do it for posts and categories, so I adopted the same approach for the sake of consistency. I am happy for another approach to be taken, though, just so long as it achieves (at least) the same thing. Feel free to write an alternative!